<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>today_patient</class>
 <widget class="QDialog" name="today_patient">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="txt_choose_patient">
     <property name="text">
      <string>选择分药的患者</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="patient_container" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>取消分药</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>today_patient</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
